Core/RoverControllerPresets/IRoverDriveController.cs

```diff
@@ -9,10 +9,15 @@ public interface IRoverDriveController
 	/// </summary>
 	public RoverControl CalculateMoveVector();
 
+	/// <summary>
+	/// Checks which kinematic mode is currently active
+	/// </summary>
+	public MqttClasses.KinematicMode CheckKinematicMode();
```

Comment:
Whattya needing function for? That can be substituted with property like this: `public KinematicMode Mode { get; }` and then for each
implementing class u have:
`public KinematicMode Mode { get; } = KinematicMode.Ackermann;`
or
`public KinematicMode Mode { get; private set; } = KinematicMode.Ackermann;`
depending on what you need

Reply:
Adressed in [92cc0a1](https://github.com/wisniax/RoverControlApp/pull/36/commits/92cc0a1eb0c72c8bd1f3ac6f9d9c636ed1b9a60a)